uavcan/dsdl/parser.py

```diff
@@ -217,6 +217,7 @@ def __init__(self, full_name, kind, source_file, default_dtid, source_text):
         self.default_dtid = default_dtid
         self.kind = kind
         self.source_text = source_text
+        self.data_type_signature = None
```

Comment:
Per PEP8, private attributes should be prefixed with an underscore: `_data_type_signature`